x-pack/legacy/plugins/alerting/server/lib/create_fire_handler.ts

```diff
@@ -9,15 +9,15 @@ import { ActionsPlugin } from '../../../actions';
 import { transformActionParams } from './transform_action_params';
 
 interface CreateFireHandlerOptions {
```

Comment:
Yeah was thinking about it. Also about how it works in the alert type executor since the terminology is a bit confusing.

Example:
```
services
    .alertInstanceFactory('1')
    .fire('default', {})
```

I'm thinking having `execute` there may cause confusion since it's not firing / executing the actions within that function per say (if it did you would `await .fire(...)`). I'm definitely open to changing it just had this concern come to mind. Thoughts?

Reply:
Maybe `enqueue` or `schedule` makes more sense?

Reply:
I like `enqueue`. I will create a card within the project to change the terminology. From a quick check, there's 109 places using the word `fire` within the `alerting` plugin that should change to `enqueue` or `execute`.